StyleCop.Analyzers/StyleCop.Analyzers.Test.CSharp11/Lightup/IImportScopeWrapperCSharp11UnitTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace StyleCop.Analyzers.Test.CSharp11.Lightup
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using Microsoft.CodeAnalysis;
+    using Moq;
+    using StyleCop.Analyzers.Lightup;
+    using StyleCop.Analyzers.Test.CSharp10.Lightup;
+    using Xunit;
+
+    public partial class IImportScopeWrapperCSharp11UnitTests : IImportScopeWrapperCSharp10UnitTests
+    {
+        [Theory]
+        [InlineData(0)]
+        [InlineData(1)]
+        [InlineData(2)]
+        public void TestCompatibleInstance(int numberOfAliasSymbols)
+        {
+            var obj = CreateImportScope(numberOfAliasSymbols);
+            Assert.True(IImportScopeWrapper.IsInstance(obj));
+            var wrapper = IImportScopeWrapper.FromObject(obj);
+            Assert.Equal(obj.Aliases, wrapper.Aliases);
```

Comment:
I updated this test to verify the actual alias symbols returned from the wrapper. Used the Moq package to be able to create IAliasSymbol instances more easily. Ok?